bambi/families/multivariate.py

```diff
@@ -6,40 +6,25 @@
 
 class MultivariateFamily(Family):
     def predict(self, model, posterior, linear_predictor):
-        """Predict mean response"""
-        mean = self.link.linkinv(linear_predictor)
-        obs_n = mean.shape[-1]
-        name = model.response.name + "_mean"
-        coord_name = name + "_dim_0"
-
-        # Drop var/dim if already present
-        if name in posterior.data_vars:
-            posterior = posterior.drop_vars(name)
+        return NotImplemented
```

Comment:
If your removing logic I would suggest a deprecation warning, or a more informative why notimplemented exception

Reply:
Actually, this was never used. There was only one child of `MultivariateFamily` (`Categorical`) and it always implemented its own `predict()` method so users will never see a problem with this `NotImplemented` stuff.

I do agree that maybe I can handle it in a better way. I haven't thought very well of other multivariate families. The only extra thing I have in mind is 'MultivariateNormal'. 

On top of that, with the addition of `MutableData` to PyMC, it does make a lot of sense to rely on PyMC for predictions instead of computing them ourselves. For example, this would simplify things a lot when Bambi includes what's known as "distributional models" (e.g. see https://cran.r-project.org/web/packages/brms/vignettes/brms_distreg.html)